sample.env

```diff
@@ -87,3 +87,7 @@ SB_MAP_PARSER_MAX_CONCURRENT=1
 
 # Datadog configuration for log monitoring. If not specified, logs will only be stored locally
 #SB_DATADOG_KEY=DEADBEEF
+
+# Path to the Google Cloud credentials file. Usually downloaded as a JSON file from Google Cloud
+# representing the service account key.
+#GOOGLE_APPLICATION_CREDENTIALS=/path/to/the/credentials.json
```

Comment:
Paths to files won't work for production deployment unless you map the file into the container, so this PR needs to have updated the various docker-compose files to do so. It would probably be good to also provide a mock API for integration tests (I assume there is a way to configure the server this client is talking to, which you will need to implement this) + enforce that this service is always configured in production, as silently not moderating images seems bad